modules/fts/pages/fts-search-response.adoc

```diff
@@ -1,14 +1,1316 @@
+    "query": {
+        "field": "country",
+        "term": "united"
+    },
+    "facets": {
+        "types": {
+            "size": 10,
+            "field": "updated",
+            "date_ranges": [
+                {
+                "name": "old",
+                "end": "2010-08-01"
+                },
+                {
+                "name": "new",
+                "start": "2010-08-01"
+                }
+            ]
+        }
+    }
+}'
+----
+
+Results:
+
+[source,json]
+----
+"facets": {
+    "types": {
+        "field": "updated",
+        "total": 954,
+        "missing": 0,
+        "other": 0,
+        "date_ranges": [
+            {
+            "name": "old",
+            "end": "2010-08-01T00:00:00Z",
+            "count": 934
+            },
+            {
+            "name": "new",
+            "start": "2010-08-01T00:00:00Z",
+            "count": 20
+            }
+        ]
+    }
+}
+----
+
+== Highlighting
+
+The `Highlight` object indicates whether highlighting was requested. 
+
+The pre-requisite includes term vectors and store options to be enabled at the field level to support Highlighting.
+
+The highlight object contains the following fields:
+
+* *style* - (Optional) Specifies the name of the highlighter. For example, "html"or "ansi".
+
+* *fields* - Specifies an array of field names to which Highlighting is restricted.
+
+=== Example 1
+
+As per the following example, when you search the content in the index, the matched content in the `address` field is highlighted in the search response.
+
+[source,console]
+----
+curl -u username:password -XPOST -H "Content-Type: application/json" \
+http://localhost:8094/api/index/travel-sample-index/query \
+-d '{
+    "explain": true,
+    "fields": [
+        "*"
+    ],
+    "highlight": {    
+      "style":"html",  
+      "fields": ["address"]
+    }, 
+    "query": {
+        "query": "address:farm"
+    }
+}'
+----
+
+==== Result
+
+[#fts_highlighting_in_address_field]
+image::fts-highlighting-in-address-field.png[,520,align=left]
+
+=== Example 2
+
+As per the following example, when you search the content in the index, the matched content in the `description` field is highlighted in the search response.
+
+[source,console]
+----
+curl -u username:password -XPOST -H "Content-Type: application/json" \
+http://localhost:8094/api/index/travel-sample-index/query \
+-d '{
+    "explain": true,
+    "fields": [
+        "*"
+    ],
+    "highlight": {    
+      "style":"html",  
+      "fields": ["description"]
+    }, 
+    "query": {
+        "query": "description:complementary breakfast"
+    }
+}'
+----
+
+==== Result
+
+[#fts_highlighting_in_description_field]
+image::fts-highlighting-in-description-field.png[,520,align=left]
+
+== Took
+
+The time taken (in nanoseconds) to complete the execution of a query.
+
+=== Example
+
+[source,json]
+----
+"total_hits": 56,
+"max_score": 0.8041158525040355,
+"took": 1449005,
+----
+
+== Hits
+
+[abstract]
```

Comment:
Abstract needs to be under the main heading

Reply:
updated